Team delete shows instructions if there are subteams

mlsteele opened the pull request:
```
$ kbu team delete cn5team2

Cannot delete team cn5team2 because it has active subteams.
Delete all of its subteams first:
- cn5team2.sub
- cn5team2.sub.bub

▶ ERROR team has active subteams
$ kbu team delete cn5team2.sub

Cannot delete team cn5team2.sub because it has active subteams.
Delete all of its subteams first:
- cn5team2.sub.bub

▶ ERROR team has active subteams
$ kbu team delete cn5team2.sub.bub
WARNING: This will destroy all data in cn5team2.sub.bub's chats, KBFS folders, and git repositories.

** if you are sure, please type: "nuke cn5team2.sub.bub" > nuke cn5team2.sub.bub
Success! Team cn5team2.sub.bub deleted.
$ kbu team delete cn5team2.sub
WARNING: This will destroy all data in cn5team2.sub's chats, KBFS folders, and git repositories.

** if you are sure, please type: "nuke cn5team2.sub" > nuke cn5team2.sub
Success! Team cn5team2.sub deleted.
$ kbu team delete cn5team2
WARNING: This will:

(1) destroy all data in cn5team2's chats, KBFS folders, and git repositories.
(2) prevent "cn5team2" from being used again as a team name.

** if you are sure, please type: "nuke cn5team2" > nuke cn5team2
Success! Team cn5team2 deleted.
$
```

patrickxb commented:
Remove (2) here, since there are no subteams.

> $ kbu team delete cn5team1
> WARNING: This will:
> 
> (1) destroy all data in cn5team1's chats and KBFS folders
> (2) do the same to any of cn5team1's subteams
> (3) prevent "cn5team1" from being used again as a team name.
> 
> ** if you are sure, please type: "nuke cn5team1" > nuke cn5team1
> Success! Team cn5team1 deleted.

mlsteele commented:
Good point. I updated the warning. The updated behavior is in the PR description.

Also deleting a subteam seemed too easy so I gave it a confirmation like root teams.

Also fixed a bug where `ListSubteams()` would return a deleted team, fixed that.
